__tests__/app.test.js

```diff
@@ -242,6 +242,16 @@ describe("app", () => {
           });
         });
     });
+    test("Status 200 - Articles have expected values", () => {
+      return request(app)
+        .get("/api/articles")
+        .expect(200)
+        .then(({ body: { articles } }) => {
+          expect(articles[0].comment_count).toEqual(2);
+          expect(articles[5].comment_count).toEqual(11);
+          expect(articles[11].comment_count).toEqual(0);
```

Comment:
Nice! If I'm being picky (it's part of my job haha) I'd say there is a more appropriate matcher than `.toEqual` to assert against primitive data types

Reply:
Ah I had it in the back of my head that toBe() is more performant, good to know as it's very easy to get in the habit of toEqualing everything!